Filters/FilterHelper.cs

```diff
@@ -0,0 +1,139 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Orchard.ContentManagement;
+using Orchard.Localization;
+using Orchard.Projections.FilterEditors.Forms;
+
+namespace Nwazet.Commerce.Filters {
+   public static class FilterHelper {
+        public enum PropertyType { StringType, NumericType };
+        public static Action<IHqlExpressionFactory> GetFilterPredicateNumeric(NumericOperator op, string property, string value, string min, string max) {
```

Comment:
I don't understand why we're taking in strings, only to immediately parse them into decimals.